fischkapp/src/components/AddNewCard/AddNewBackCard.tsx

```diff
@@ -29,16 +29,17 @@ export const AddNewBackCard = ({
   onSetNewCard,
 }: BackCardProps) => {
   const [backText, setBackText] = useState("");
-
   const addNewFlashCard = (frontText: string, backText: string) => {
     const newFlashCard: FlashCardType = {
-      id: 1,
+      id: flashCards.length + 1,
```

Comment:
**nitpick:** You would rather get last card id, and increment its value instead of setting it by card amount. This solution is quite dangerous, as you can have duplicate IDs. My suggestion is to use `Date.now()`, as it returns current moment timestamp as `integer`, so it's aways unique:

```js
const newFlashCard: FlashCardType = {
      id: Date.now()
``